extension/index.azureFunction.js

```diff
@@ -0,0 +1,50 @@
+import utils from './src/utils.js'
+import paymentHandler from './src/paymentHandler/payment-handler.js'
+import { getAuthorizationRequestHeader } from './src/validator/authentication.js'
+
+const FAILED_VALIDATION = 'FailedValidation'
+
+function handleSuccessResponse(context, paymentResult) {
```

Comment:
Looks like it's not always Success response?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/993/commits/01d0148e062aa412679216055d8cf4ac9d78c53e